symphony-bdk-bom/build.gradle

```diff
@@ -16,7 +16,7 @@ repositories {
 
 dependencies {
     // import Spring Boot's BOM
-    api platform('org.springframework.boot:spring-boot-dependencies:2.6.4')
+    api platform('org.springframework.boot:spring-boot-dependencies:2.6.5')
     // import Jackson's BOM
     api platform('com.fasterxml.jackson:jackson-bom:2.13.2')
```

Comment:
we can also upgrade jackson to address the latest CVE: https://github.com/FasterXML/jackson-databind/issues/2816